lib/MetaCPAN/Document/File/Set.pm

```diff
@@ -499,28 +499,47 @@ sub autocomplete_using_suggester {
         }
         );
 
-    my @docs
-        = map { $_->{text} } @{ $suggestions->{documentation}[0]{options} };
+    my %docs;
+    for my $d ( @{ $suggestions->{documentation}[0]{options} } ) {
+        next if exists $docs{ $d->{text} };
+        $docs{ $d->{text} } = $d->{score};
+    }
+
+    my @docs_uniq = sort { $docs{$b} <=> $docs{$a} } keys %docs;
```

Comment:
that's a redundant leftover from before adding the script ... will remove
I was looking for the reason it was getting low-quality results, and thought the `size => 10` which is clearly why we got random 10 matches from 'all' doc matches might be affected by the order... not smart :)